.github/README-exec/torch.readme.md

```diff
@@ -1,11 +1,35 @@
 # CLIPTorchEncoder
 
-`CLIPTorchEncoder` serve OpenAI released [CLIP](https://github.com/openai/CLIP) models with PyTorch runtime. 
-It embeds documents using either the text or the visual part of CLIP, depending on their content.
+**CLIPTorchEncoder** is the executor implemented by [clip-as-service](https://github.com/jina-ai/clip-as-service). 
+It serves OpenAI released [CLIP](https://github.com/openai/CLIP) models with PyTorch w/o JIT runtime. 
+The introduction of the CLIP model [can be found here](https://openai.com/blog/clip/).
+
+- 🔀 **Automatic**: Auto-detect image and text documents depending on their content.
+- ⚡ **Efficiency**: Non-blocking duplex streaming on requests and responses. 
+- 📈 **Observability**: Monitoring the serving via Prometheus and Grafana.
+
+
+## Model support
+
+Open AI has released 9 models so far. `ViT-B/32` is used as default model in all runtimes. Please also note that **different model give different size of output dimensions**. This will affect your downstream applications. For example, switching the model from one to another make your embedding incomparable, which breaks the downstream applications. Here is a list of supported models of each runtime and its corresponding size:
```

Comment:
```suggestion
Open AI has released 9 models so far. `ViT-B/32` is used as the default model in all runtimes. Please also note that **different models give the different sizes of output dimensions**. This will affect your downstream applications. For example, switching the model from one to another make your embedding incomparable, which breaks the downstream applications. Here is a list of supported models of each runtime and its corresponding size:
```